doc/history.rst

```diff
@@ -4,6 +4,14 @@ History
 Releases
 --------
 
+Version 4.5
+```````````
+
+* :class:`~.spawn` now has a ``use_poll`` parameter. This allows the use
+  of select.poll() on fds
+* :class:`~.fdspawn` now has a ``use_poll`` paramneter. This allow the use
+  of select.poll() on fds
```

Comment:
Let's combine these two notes into one (it's one conceptual change, even if it affects two classes) and mention that:

- It lets you get round problems with file descriptors > 1024.
- It's not used by default because of compatibility concerns, so the downstream code has to decide to use it.

Reply:
Ping @cooperlees ? If you can make these changes to the docs, then I think the PR is ready to merge.